crates/runtime/src/lib.rs

```diff
@@ -397,6 +397,16 @@ impl_runtime_apis! {
 
             (messages_to_l1, messages_to_l2)
         }
+
+        fn l1_nonce_unused(nonce: u64) -> bool {
```

Comment:
I would rather have the conversion being done outside of the runtime.
Your signature should be `fn l1_nonce_unused(nonce: Nonce) -> bool`

Reply:
Sure!